internal/kafka/internal/config/kafka.go

```diff
@@ -38,6 +39,7 @@ func NewKafkaConfig() *KafkaConfig {
 		KafkaCapacityConfigFile:        "config/kafka-capacity-config.yaml",
 		KafkaLifespan:                  NewKafkaLifespanConfig(),
 		Quota:                          NewKafkaQuotaConfig(),
+		BrowserUrl:                     "https://console.redhat.com/",
```

Comment:
I wonder if we can use `http://localhost:8080/` as default? I believe the [UI](https://github.com/bf2fc6cc711aee1a0c2a/kafka-ui) can be run locally.

Reply:
@machi1990 have you done this by any chance? :thinking:

Reply:
No, this is not something I’ve done but I am sure that’s how the UI team is doing their devs. Maybe @wtrocki would have some input